torch/_inductor/scheduler.py

```diff
@@ -2845,6 +2854,8 @@ def has_shared_data_after_reordering_loop(
         to be compatibile with node1 if that's more efficient.
         """
 
+        node1, node2 = node2, node1
```

Comment:
b) reorders the loop while also considering the other reduction ops?

Reply:
With a) and b), the generated kernel looks optimal:

Test case:
```
def test(x):
    y = torch.sum(x)
    z = x / 10.0
    z_t = z.t().contiguous().t()
    return y, z, z_t
```

Outputs:
```
0.107ms         0.201 GB         1883.86GB/s     triton_red_fused_clone_div_sum_0
0.002ms         0.000 GB            6.83GB/s     triton_red_fused_sum_1
SUMMARY (/tmp/torchinductor_shuqiyang/tmpnmzpylwk/nh/cnhqyu44azg4rc6in5vzvqyk3ouuk6dzyhkbaiv3jrifejoxhbag.py)
0.11ms           0.20 GB         1842.66GB/s
```
Kernel: P1670240539